src/scipp/compat/xarray_compat.py

```diff
@@ -79,6 +123,46 @@ def from_xarray_dataarray(da: xr.DataArray) -> DataArray:
                      name=da.name or "")
 
 
+def to_xarray_dataarray(da: DataArray) -> xr.DataArray:
+    """Converts a scipp.DataArray object to an xarray.DataArray object.
+
+    Parameters
+    ----------
+    da:
+        A DataArray object to be converted.
+
+    Returns
+    -------
+    :
+        The converted xarray.DataArray object.
+
+    See Also
+    --------
+    scipp.compat.to_xarray_dataarray, scipp.compat.from_xarray_dataset
+    """
+
+    import xarray as xr
+    data = _var_to_xarray(da.data)
+    coords = {}
+    for key, coord in da.coords.items():
+        if da.coords.is_edges(key):
+            raise ValueError("Xarray does not support coordinates with bin edges.")
```

Comment:
Above for variance we just warn, here we raise? Maybe we need to support a dict of policies?

```python
scipp.compat.to_xarray(da, policies={'variance'='warn', 'bin_edges'='midpoints'})
```

Reply:
The API for the policies might be something that would take a while to figure out? For instance, we could envisage supporting masks by merging all scipp masks into one, and then making the data in the xr.DataArray a masked numpy array?

Maybe we can add the policies in a later PR, if it is needed?

Reply:
👍